src/resolversFactory.js

```diff
@@ -0,0 +1,50 @@
+var util = require('util');
+var http = require('http');
+
+function createWofPipResolver(url) {
```

Comment:
if you name this parameter baseUrl instead of url you can get rid of line 5 and avoid having the url variable defined on line 8 shadow this parameter